### ISSUE TITLE
role jsx-a11y/no-noninteractive-element-interactions triggers throws with `onLoad` on <img />

### ISSUE THREAD

**Issue by NehrDani** [evcohen/eslint-plugin-jsx-a11y#467]
The rule throws `jsx-a11y/no-noninteractive-element-interactions` when using `onLoad` on an `img` tag.

```js
<img onLoad={() => console.log('loaded')} />
```

I don't this makes sense :thinking: because it's not an user interaction.

Using:
- `eslint v5.1.0`
- `eslint-plugin-jsx-a11y v.6.1.1`

**Comment by jessebeach** [comment]
@NehrDani agree. Would you like to post a PR to exclude this attribute in the rule?

**Comment by NehrDani** [comment]
> Would you like to post a PR to exclude this attribute in the rule?

I can try. I'm not that deep into these rules.

**Comment by NehrDani** [comment]
I really need to apologize. It was my fault. The rules are correct (no fix needed). I overwrite most rules, that's why I got this. Maybe I should use the recommended config instead.
This can be closed.

**Comment by jessebeach** [comment]
@NehrDani no worries! Thank you for taking the time to report an issue and helping us keep the quality high. I appreciate the effort.
